mop/brokers/asassn.py

```diff
@@ -0,0 +1,224 @@
+import os
+from django import forms
+import requests
+import urllib
+from urllib.error import HTTPError
+from urllib.request import Request, urlopen
+from astropy.time import Time, TimezoneInfo
+import datetime
+import pandas as pd
+import lxml.html as lh
+import mechanicalsoup
+from bs4 import BeautifulSoup
+from astropy.coordinates import SkyCoord
+import astropy.units as unit
+import json
+from html_table_parser.parser import HTMLTableParser
+
+
+from tom_targets.models import Target
+from tom_alerts.alerts import GenericBroker, GenericQueryForm
+from tom_dataproducts.models import ReducedDatum
+
+BROKER_URL = 'http://www.astronomy.ohio-state.edu/asassn/transients.html' #should be a known variable?
+photometry = 'https://asas-sn.osu.edu/photometry'
+
+
+class ASASSNBroker():
+    def __init__(self, name):  
+        self.name = name 
+    #name='ASAS-SN'
+
+    def open_webpage(self):
+        page_status_code=requests.get(BROKER_URL).status_code
+        return page_status_code
+
+    def retrieve_transient_table(self):
+        page=requests.get(BROKER_URL)
+        doc = lh.fromstring(page.content)
+        tr_elements = doc.xpath('//tr')
+        col=[]
+        i=0
+        for t in tr_elements[0]:
+            i+=1
+            content=t.text_content()
+            col.append((content,[]))
+        for j in range(1,len(tr_elements)):
+            #T is our j'th row
+            T=tr_elements[j]
```

Comment:
This is overalll good, but can you avoid giving variables such name (T). Could you modify this before I accept?

Reply:
Fixed!